File: Easy/26/26.java
class Solution {
    public int removeDuplicates(int[] nums) {
        int length = nums.length;
        if (length == 0)
            return 0;
        int count = 0;
        for (int idx = 1; idx < length; ++idx){
            if (nums[idx] != nums[count]){
                ++count;
                nums[count] = nums[idx];
            }
        }

        return count + 1;
    }
}

File: Easy/26/26.cpp
#include <iostream>
#include <vector>

class Solution {
public:
    int removeDuplicates(std::vector<int>& nums) {
        if (nums.size() == 0)
            return 0;
        std::vector<int>::iterator iter;
        int current_value = nums[0], count = 1;
        for(iter = nums.begin(); iter != nums.end(); ++iter){
            if (*iter != current_value){
                current_value = *iter;
                nums[count] = current_value;
                ++count;
            }
        }

        return count;
    }
};
